modules/Bio/EnsEMBL/Analysis/Hive/Config/production_rnaseq_db.pm

```diff
@@ -0,0 +1,663 @@
+                                ' -dbname '.$self->o('rnaseq_db','-dbname').
+                                ' -prod_dbuser '.$self->o('user_r').
+                                ' -prod_dbhost '.$self->o('production_db','-host').
+                                ' -prod_dbname '.$self->o('production_db','-dbname').
+                                ' -prod_dbport '.$self->o('production_db','-port').
+                                ' -nodaf -ise'
+                       },
+        -max_retry_count => 0,
+        -rc_name => '8GB',
+        -flow_into => {
+                        1 => ['null_rnaseq_columns'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'null_rnaseq_columns',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+        -parameters => {
+          db_conn => $self->o('rnaseq_db'),
+          sql => [
+            'UPDATE dna_align_feature SET external_db_id = NULL',
+          ],
+        },
+        -rc_name    => 'default',
+        -flow_into => {
+                        1 => ['rnaseq_set_align_type'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'rnaseq_set_align_type',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+        -parameters => {
+          db_conn => $self->o('rnaseq_db'),
+          sql => [
+            'UPDATE dna_align_feature SET align_type = "ensembl"',
+          ],
+        },
+        -rc_name    => 'default',
+        -flow_into => {
+                        1 => ['rnaseq_gene_sanity_checks'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'rnaseq_gene_sanity_checks',
+        -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAnalysisSanityCheck',
+        -parameters => {
+                         target_db => $self->o('rnaseq_db'),
+                         sanity_check_type => 'gene_db_checks',
+                         min_allowed_feature_counts => get_analysis_settings('Bio::EnsEMBL::Analysis::Hive::Config::SanityChecksStatic',
+                                                                             'gene_db_checks')->{'rnaseq_final'},
+                       },
+
+        -rc_name    => '4GB',
+        -flow_into => {
+                        1 => ['create_bam_file_job'],
+                      },
+      },
+
+
+      {
+        -logic_name => 'create_bam_file_job',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::JobFactory',
+        -parameters => {
+          inputcmd => 'cd #working_dir#; ls *.bam',
+          column_names => ['bam_file'],
+          working_dir => $self->o('merge_dir'),
+        },
+        -rc_name => 'default',
+        -flow_into  => {
+          '2->A' => ['bam2bigwig'],
+          'A->1' => ['concat_md5_sum'],
+        },
+      },
+
+
+      {
+        -logic_name => 'bam2bigwig',
+        -module => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -parameters => {
+	  deeptools_bamcoverage => $self->o('deeptools_bamcoverage_path'),
```

Comment:
I know there are real tools for that and I should have set them up correctly but I should have also mentioned it in the tickets. Could you please remove the tab and use space for the indentation